src/tezos_rpc/http.ml

```diff
@@ -0,0 +1,41 @@
+open Helpers
+open Error
+
+type 'a method_ =
```

Comment:
I don't see any test module, is it normal?

Reply:
Currently we don't have a lot of automated tests, and we're in crunch mode, but @d4hines is working on having more test infrastructure.

Reply:
What about basic testing?
For instance, even if we can suppose that `yojson` is working, having tests like:
> when calling to_yojson on this object, it produces the following JSON

or also

> when calling to_yojson then from_yojson, it gives back the original object

If we always say "we can do that later" we will never do it... Moreover, some tests are just quick-win and ensure our domain is safe in the real world.

For instance, I think the function `http_request` isn't so difficult to test, am I wrong?
Or maybe we can start by only testing `http_post` function, which is exposed and used, and test the others later.
What do you think?

Reply:
Opened #454